src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,26 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.service.suppliers.FigureSupplier;
+import java.util.Random;
+
 public class HelloWorld {
+    private static FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[new Random().nextInt(6)];
```

Comment:
lets make constant for 6
you dont ave to randomize the size of the array - it should be 6